pipeline/notifications/check_message_test.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+
+import pytest
+
+from pipeline.notifications import Notifier
+from pipeline.types import ChatData, Match, Message, Notification, User
+
```

Comment:
Good tests! Would recommend adding a case where both users have sent messages, so there are no users to notify.